src/js/features/repoLookup/usernameInput/usernameInput.component.js

```diff
@@ -0,0 +1,31 @@
+import { action, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import repoLookupStore from '../repoLookup.store';
+
+export default @observer class UsernameInputComponent extends React.Component {
+    @observable input = '';
```

Comment:
This is interesting. Does the React component re-render when this value changes? I bet it does because of @observer. I wonder how the mobx people implemented it and if it is using some version of `this.state + this.setState()`